starter-code/iteration3/src/app/nba-champions/nba-champions.component.html

```diff
@@ -0,0 +1,12 @@
+<h1>NBA Champions</h1>
+<ul>
+    <li *ngFor="let champion of nbaChampions">
+        <h1>  {{ champion.year }} </h1>
+        <h1>  {{ champion.team }} </h1>
+        <img src = '{{champion.logo}}'>
+        <small> {{ animal.category }} </small>
```

Comment:
This line is leftover from the code you were referring to when you did this exercise. Delete it since it's breaking your code.